src/vivarium_gates_iv_iron/components/observers.py

```diff
@@ -52,10 +58,11 @@ def get_state_function(state: Union[str, bool, List]) -> Callable:
                 stratification_key: get_state_function(source_value)
                 for stratification_key, source_value in categories.items()
             }
-            if is_pipeline:
-                self.pipelines[source_name] = builder.value.get_value(source_name)
-            else:
-                columns_required.append(data_keys.WASTING.name)
+
```

Comment:
Why is this commented out?

Edit: I just realized this is a bug. The else block should read `columns_required.append(source_name)`, but you shouldn't have this commented.